helix-term/src/keymap.rs

```diff
@@ -169,6 +188,14 @@ impl KeyTrieNode {
         &self.name
     }
 
+    pub fn len(&self) -> usize {
+        self.map.len() + self.fallback.is_some() as usize
+    }
+
+    pub fn is_empty(&self) -> bool {
+        self.len() == 0
```

Comment:
Don't we need to check `fallback` here too?

Reply:
That's handled by `len`.